client/termsetting/termsetting.ts

```diff
@@ -874,7 +878,19 @@ export async function fillTermWrapper(tw: TermWrapper, vocabApi: VocabApi, defau
 	await call_fillTW(tw, vocabApi, defaultQByTsHandler)
 
 	mayValidateQmode(tw)
-	return tw as TermWrapper
+}
+
+export async function fillTwLst(twlst: TermWrapper[], vocabApi: VocabApi, defaultQByTsHandler?: DefaultQByTsHandler) {
+	const ids: Array<string | undefined> = []
+	for (const tw of twlst) ids.push(tw.id)
+	const terms = await vocabApi.getTerms(ids)
+
```

Comment:
@airenzp In the single `fillTermWrapper()`, there is the code below. Why is that check not in the `fillTwLst()`? Just making sure that necessary validation steps are not removed in the new function.
```js
if (!tw.term) {
		if (tw.id == undefined || tw.id === '') throw 'missing both .id and .term'
		// has .id but no .term, must be a dictionary term
		// as non-dict term must have tw.term{}
		tw.term = await vocabApi.getterm(tw.id)
	}
```

Reply:
Yeah, fixed now.